website/src/components/AdoptersSlider/index.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { makeStyles } from "@material-ui/core/styles";
+import Carousel from "../Carousel";
+
+const useStyles = makeStyles((theme) => ({
+  logoCarousel: {
+    marginTop: theme.spacing(6),
+  },
+}));
+
+const AdopterSlider: React.FC = () => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  var logoSliderSettings = {
+    dots: false,
+    autoplay: true,
+    autoplaySpeed: 0,
+    speed: 10000,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    variableWidth: true,
```

Comment:
All the logo widths are same

Reply:
it is given to achive the smooth scroll effect on slider, by giving `variableWidth:true` we can show any no of slides without worrying about the overlap of slides

Reply:
Ok